django/db/models/functions/math.py

```diff
@@ -158,9 +158,23 @@ def get_group_by_cols(self, alias=None):
         return []
 
 
-class Round(Transform):
+class Round(FixDecimalInputMixin, Transform):
     function = 'ROUND'
     lookup_name = 'round'
+    arity = None  # Override Transform's arity=1 to enable passing precision.
+
+    def __init__(self, expression, precision=0, **extra):
+        super().__init__(expression, precision, **extra)
+
+    def as_sqlite(self, compiler, connection, **extra_context):
+        precision = self.get_source_expressions()[1]
+        if isinstance(precision, Value) and precision.value < 0:
+            raise ValueError('Backend does not support negative precision.')
```

Comment:
`Backend` supports negative precision, `SQLite` does not:
```suggestion
            raise ValueError('SQLite does not support negative precision.')
```